src/app/(proper_react)/(redesign)/(public)/LandingView.test.tsx

```diff
@@ -164,6 +173,61 @@ describe("When Premium is not available", () => {
     expect(signIn).toHaveBeenCalledTimes(1);
   });
 
+  it("displays a banner announcing the rebrand to 'Mozilla Monitor' by default", () => {
+    const ComposedDashboard = composeStory(LandingNonUs, Meta);
+    render(<ComposedDashboard />);
+
+    const rebrandAnnouncement = screen.getByText(
+      "New name, look and even more ways to",
+      { exact: false },
+    );
+
+    expect(rebrandAnnouncement).toBeInTheDocument();
+  });
+
+  it("hides the banner announcing the rebrand to 'Mozilla Monitor' after clicking the dismiss button", async () => {
+    const ComposedDashboard = composeStory(LandingNonUs, Meta);
+    render(<ComposedDashboard />);
+
+    const rebrandAnnouncement = screen.getByText(
+      "New name, look and even more ways to",
+      { exact: false },
+    );
+    const dismissButton = within(rebrandAnnouncement.parentElement!).getByRole(
```

Comment:
I was actually hoping to not have to use it, but I didn't have a good way to target the complete banner. (I think it might need some accessible role, but I'm not sure which...)

Reply:
Maybe, I'm not sure. The general idea is to make our tests perform much the same actions as our users would do, and looking at the string "New name, look, etc" is obviously closer to that than looking at the source code and finding a test ID would be. But: looking for that string, then inspecting the DOM, looking for the parent element, and then finding a button in there of course _also_ isn't really what a user would do.

But with the response I got from the a11y team, I'm now simulating a user using landmark navigation to get to the `<aside>`, and then seeing the text in there: 9cac3055e. That might sort of be better?